libmultilabel/data_utils.py

```diff
@@ -48,7 +48,7 @@ def generate_batch(data_batch):
     }
 
 
-def get_dataset_loader(config, data, word_dict, classes, shuffle=False, train=True):
+def get_dataset_loader(config, data, word_dict, classes, device, shuffle=False, train=True):
```

Comment:
Why not remove `config` here?
Also we don't need `train` if `batch_size if passed.

Reply:
Sure. Maybe we can start from taking `train` out.
--
I've had a discussion with Henry and decide to make the change of this function in the next PR. 
(1) long arguments  for `train_loader`,  `val_loader`, and `test_loader` is one thing
(2) should `TextDataset` be in this function's another